test/e2e/framework/helper/certificates.go

```diff
@@ -248,6 +248,13 @@ func (h *Helper) ValidateIssuedCertificate(certificate *cmapi.Certificate, rootC
 		}
 	}
 
+	if cond := apiutil.GetCertificateCondition(certificate, cmapi.CertificateConditionReady); cond.Status != cmmeta.ConditionTrue ||
+		cond.ObservedGeneration != certificate.Generation {
```

Comment:
nit: can this be split out into one line that declares `cond` above? Wrapping this over two lines makes it harder to consider/understand 😬

Reply:
Agreed, it would make a timy bit more readable to have it as James said:

```go
cond := apiutil.GetCertificateCondition(certificate, cmapi.CertificateConditionReady)
if cond.Status != cmmeta.ConditionTrue || cond.ObservedGeneration != certificate.Generation {
}

```